### PR TITLE
Fix footer nested link

### PR DESCRIPTION
**Pull request by SSK-14**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

Fix the nested link in the footer.
Replaced the link tag with p tag and added href to tech stacks.

Fixes # (627)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Tested in my local env.

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by atapas** [comment]
@Angryman18 Can you please review this one?

File: src/common/footer/Footer.jsx [hunk 1]
```diff
@@ -5,14 +5,14 @@ const Footer = () => {
     <footer className="app-footer text-center">
       <hr className="separater" />
       <div className="flex justify-center items-center py-2">
-        <Link to="/tech-stacks">
+        <p>
```
Comment (Angryman18) [inline src/common/footer/Footer.jsx:8]: it seems thing got reverted. From my view we should be only using Components coming from react-router-dom in order to do routing which prevent the default browser behaviour. unless Link component its fine to use `href` attribute but its inconsistent and making page refresh which is not an improvement from my prospective.

Comment (SSK-14) [reply src/common/footer/Footer.jsx:8]: Yes i accept your point @Angryman18 . Then better to add Link tag instead of the anchor tag ?

**Comment by Angryman18** [comment]
@atapas can we get it deployed

**Comment by atapas** [comment]
> @atapas can we get it deployed

Done

**Review by Angryman18 — APPROVED** [review]

**Review by atapas — APPROVED** [review]
Looks good to me.

**Comment by atapas** [comment]
@all-contributors please add @SSK-14 for Code

**Pull request merged**